pkg/model/update_settings.go

```diff
@@ -23,6 +29,26 @@ func (us UpdateSettings) WithMaxParallelUpdates(n int) UpdateSettings {
 	return us
 }
 
+func (us UpdateSettings) K8sUpsertTimeout() time.Duration {
```

Comment:
again, made getters/setters for private field to make sure we can never call upsert with a nonsense timeout. happy to hear that this is overkill and change